vividus-extension-selenium/build.gradle

```diff
@@ -3,6 +3,7 @@ project.description = 'Vividus extension for selenium'
  ext {
      seleniumVersion = '4.1.1'
      browserupProxyVersion = '2.1.2'
+     nettyVersion = '4.1.70.Final'
```

Comment:
https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-web-app/build.gradle#L5

Reply:
also should the same dependencies be removed from `vividus-plugin-web-app`?